src/game/laramisc.c

```diff
@@ -14,10 +14,19 @@
 #include "global/vars.h"
 #include "specific/sndpc.h"
 #include "util.h"
+#include "game/text.h"
 
 #include <stddef.h>
 #include <stdint.h>
 
+int32_t lara_jump_error = 0;
+TEXTSTRING *LaraText = NULL;
+TEXTSTRING *LaraText2 = NULL;
+
+PHD_3DPOS_F lara_float_pos;
+double lara_fall_speed_f = 0.0;
+double lara_speed_F = 0.0;
```

Comment:
PascalCase for all of the above variables

Consider moving these to `ITEM_INFO`, it's not like we need to worry about the game using these structures as well – everything was decompiled anyway

Reply:
From memory the ITEM_INFO are loaded or partially loaded, but modifying it did case crashes or issues,